src/schema/sellable.ts

```diff
@@ -25,6 +27,10 @@ export const Sellable = new GraphQLInterfaceType({
     },
     price: {
       type: GraphQLString,
+      deprecationReason: deprecate({
+        inVersion: 2,
+        preferUsageOf: "sale_message",
```

Comment:
What's the context behind this? I'm assuming this is just a known change that I'm unaware of.

Reply:
The `EditionSet` type has this field as being deprecated, which means that when it goes away it no longer conforms to this interface. At the time I checked usage of the others and I didn’t spot any, in Emission at least, so I went ahead and deprecated it in the interface and the other type that conforms to it (Artwork iirc). If in the future it turns out we still need it we can always add it back to v2.